src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,25 +1,1002 @@
+    { 111, 37, 22 },  { 134, 37, 22 },  { 157, 37, 22 },  { 42, 51, 22 },   { 65, 51, 22 },   { 88, 51, 22 },
+    { 111, 51, 22 },  { 134, 51, 22 },  { 157, 51, 22 },  { 42, 65, 22 },   { 65, 65, 22 },   { 88, 65, 22 },
+    { 111, 65, 22 },  { 134, 65, 22 },  { 157, 65, 22 },  { 42, 80, 22 },   { 65, 80, 22 },   { 88, 80, 22 },
+    { 111, 80, 22 },  { 134, 80, 22 },  { 157, 80, 22 },  { 202, 37, 14 },  { 215, 37, 14 },  { 229, 37, 14 },
+    { 243, 37, 14 },  { 257, 37, 14 },  { 271, 37, 14 },  { 202, 51, 14 },  { 215, 51, 14 },  { 229, 51, 14 },
+    { 243, 51, 14 },  { 257, 51, 14 },  { 271, 51, 14 },  { 202, 65, 14 },  { 215, 65, 14 },  { 229, 65, 14 },
+    { 243, 65, 14 },  { 257, 65, 14 },  { 271, 65, 14 },  { 202, 80, 14 },  { 215, 80, 14 },  { 229, 80, 14 },
+    { 243, 80, 14 },  { 257, 80, 14 },  { 271, 80, 14 },  { 42, 111, 14 },  { 54, 111, 14 },  { 66, 111, 14 },
+    { 78, 111, 14 },  { 114, 99, 14 },  { 142, 99, 14 },  { 188, 99, 14 },  { 202, 99, 14 },  { 216, 99, 14 },
+    { 230, 99, 14 },  { 244, 99, 14 },  { 258, 99, 14 },  { 188, 113, 14 }, { 202, 113, 14 }, { 216, 113, 14 },
+    { 230, 113, 14 }, { 244, 113, 14 }, { 258, 113, 14 }, { 272, 113, 14 }, { 54, 141, 14 },  { 112, 134, 14 },
+    { 155, 134, 14 }, { 194, 130, 32 }, { 194, 144, 32 }, { 262, 134, 14 }, { 43, 168, 14 },  { 55, 168, 14 },
+    { 67, 168, 14 },  { 79, 168, 14 },  { 127, 168, 14 }, { 139, 168, 14 }, { 151, 168, 14 }, { 163, 168, 14 },
+    { 202, 168, 22 }, { 223, 168, 22 }, { 244, 168, 22 }, { 265, 168, 22 }, { 42, 202, 14 },  { 73, 202, 14 },
+    { 136, 202, 38 }, { 214, 202, 74 }, { 214, 202, 74 },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_kaleido_scope/func_80819238.s")
+s16 sSlotItems[] = {
+    // Items Row 1
+    ITEM_OCARINA,
+    ITEM_BOW,
+    ITEM_ARROW_FIRE,
+    ITEM_ARROW_ICE,
+    ITEM_ARROW_LIGHT,
+    ITEM_OCARINA_FAIRY,
+    // Items Row 2
+    ITEM_BOMB,
+    ITEM_BOMBCHU,
+    ITEM_STICK,
+    ITEM_NUT,
+    ITEM_MAGIC_BEANS,
+    ITEM_SLINGSHOT,
+    // Items Row 3
+    ITEM_POWDER_KEG,
+    ITEM_PICTO_BOX,
+    ITEM_LENS,
+    ITEM_HOOKSHOT,
+    ITEM_SWORD_GREAT_FAIRY,
+    ITEM_LONGSHOT,
+    // Items Row 4
+    ITEM_BOTTLE,
+    ITEM_POTION_RED,
+    ITEM_POTION_GREEN,
+    ITEM_POTION_BLUE,
+    ITEM_FAIRY,
+    ITEM_MUSHROOM,
+    // Masks Row 1
+    ITEM_MASK_POSTMAN,
+    ITEM_MASK_ALL_NIGHT,
+    ITEM_MASK_BLAST,
+    ITEM_MASK_STONE,
+    ITEM_MASK_GREAT_FAIRY,
+    ITEM_MASK_DEKU,
+    // Masks Row 2
+    ITEM_MASK_KEATON,
+    ITEM_MASK_BREMEN,
+    ITEM_MASK_BUNNY,
+    ITEM_MASK_DON_GERO,
+    ITEM_MASK_SCENTS,
+    ITEM_MASK_GORON,
+    // Masks Row 3
+    ITEM_MASK_ROMANI,
+    ITEM_MASK_CIRCUS_LEADER,
+    ITEM_MASK_KAFEIS_MASK,
+    ITEM_MASK_COUPLE,
+    ITEM_MASK_TRUTH,
+    ITEM_MASK_ZORA,
+    // Masks Row 4
+    ITEM_MASK_KAMARO,
+    ITEM_MASK_GIBDO,
+    ITEM_MASK_GARO,
+    ITEM_MASK_CAPTAIN,
+    ITEM_MASK_GIANT,
+    ITEM_MASK_FIERCE_DEITY,
+};
```

Comment:
Errr... this looks like a generic list that belongs with kaleido_scope or kaleido_mask, instead of debug?

Reply:
You'd think, yet it's only used in the debug menu here, plus ordering of data matters, so it has to be in this file.